### PR TITLE
CAMEL-18957: converted camel-cassandra to camel-test-infra-core

### PR DESCRIPTION
**Pull request by orpiske**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]
Just a minor finding.

File: components/camel-cassandraql/src/test/java/org/apache/camel/component/cassandra/integration/BaseCassandra.java [hunk 1]
```diff
@@ -23,20 +23,35 @@
...
     public static CassandraLocalContainerService service;
 
+    @Order(2)
+    @RegisterExtension
+    public static CamelContextExtension camelContextExtension = new DefaultCamelContextExtension();
+
     public static final String KEYSPACE_NAME = "camel_ks";
     public static final String DATACENTER_NAME = "datacenter1";
 
+    protected CamelContext context = camelContextExtension.getContext();;
```
[the remark this answers is not in the document]
Comment (orpiske) [reply components/camel-cassandraql/src/test/java/org/apache/camel/component/cassandra/integration/BaseCassandra.java:53]: Ouch, good catch! Let me fix it. Thanks!!!

**Comment by orpiske** [comment]
Marking as draft for now. I need to review some results from my environment.

**Comment by orpiske** [comment]
> Marking as draft for now. I need to review some results from my environment.

Looking good, merging.

**Pull request merged**